src/stat.cc

```diff
@@ -724,8 +724,8 @@ DumpInfo(Mgr::InfoActionData& stats, StoreEntry* sentry)
     storeAppendPrintf(sentry, "\tTotal accounted:       %6.0f KB\n",
                       stats.total_accounted / 1024);
     {
-        MemPoolGlobalStats mp_stats;
-        memPoolGetGlobalStats(&mp_stats);
+        Mem::PoolStats mp_stats;
+        Mem::GlobalStats(mp_stats); // XXX: pointless lookup ?
```

Comment:
```suggestion
        Mem::GlobalStats(mp_stats); // XXX: pointless lookup
```

Reply:
Are you sure of that being pointless (eg no side effects of rebuilding TheMeter or such)?

If yes, then I will drop the lookup instead of leaving the XXX question.

Reply:
* The mp_stats lookup itself (i.e. filling mp_stats with values) is pointless because mp_stats is not used.
* Mem::GlobalStats() has side effects; those side effects might be important in this DumpInfo() context. They should be removed, but I have not studied all the details to suggest a specific fix.
* DumpInfo() should not have side effects like those injected by Mem::GlobalStats(), but something else may need to "flush" statistics before or after DumpInfo() is called (in some cases). I did not study all the necessary details to suggest a specific fix.

Due to complications in the second and third bullets, I do not recommend removing this call in this PR. Please adjust the XXX text if you think the current version needs adjustment (or leave it as is if you think it does not).